src/Hangman/Domain/Repositories/GamesRepository.ts

```diff
@@ -1,12 +1,42 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, Logger } from '@nestjs/common';
 import { GameDto } from '../../Infrastructure/Dto/Game.dto';
 import { Game } from '../AggregateRoot/Game.aggregate';
+import { EventStore } from '@berniemac/event-sourcing-nestjs';
 
 @Injectable()
 export class GamesRepository {
+  constructor(private readonly eventStore: EventStore) {}
+  private logger = new Logger(GamesRepository.name);
+
+  async findOneById(aggregateId: string): Promise<Game> {
+    const game = new Game(aggregateId);
+    const eventHistory = await this.eventStore.getEvents('game', aggregateId);
+
+    // how is this performant in a big application?
```

Comment:
// how is this performant in a big application?
// for every action, find all events on the aggregate
// replay them then do the action.
// Name change on a aggregate that has 1000 events:
// Replay all 1000 events, then change the name
// Next change, again all 1001 events. etc